src/components/Checkbox/Checkbox.mdx

```diff
@@ -0,0 +1,26 @@
+import { Preview, Story, ArgsTable, PRIMARY_STORY } from '@storybook/addon-docs/blocks';
+import Checkbox from './Checkbox.vue';
+
+# Checkbox
+
+A checkbox represents a value in typically a form that can either be true or false.
+
+<Preview>
+  <Story id="components-checkbox--primary" />
+</Preview>
+
+## How to Use
+
+The label and v-model are required. The other props are optional.
+
+```html
+<Checkbox
+  label="I am a checkbox"
+  v-model="mySwitch"
+>
+</Checkbox>
+```
```

Comment:
I would love to see a bit more documentation in general about how to use. For example, when I tried using it, I tried to pass in `value` but that didn't work, and had to go digging around to figure out that I had to pass in `nativeValue` instead. Also would love some documentation/explanation of the `trueValue`/`falseValue`, because I think that's really neat!